LanguageFeatures/Static-access-shorthand/constant_expression_A01_t03.dart

```diff
@@ -29,10 +29,11 @@ class C {
 
 extension type const ET(String value) {
   const ET.id(this.value, [String s = ""]);
-  const factory ET.f(String value, {String s = ""}) = ET;
+  const ET.id2(this.value, {String s = ""});
+  const factory ET.f(String value, {String s}) = ET.id2;
 }
 
-main() {
+main() {/*
```

Comment:
Is the line range 36-97 intended to be commented out? Looks like a temporary modification that wasn't undone.

Reply:
Of course. Thank you. Fixed.

Reply:
Thanks!